lib/components/phone_number_field.dart

```diff
@@ -119,16 +127,20 @@ class _PhoneNumberFormFieldState extends State<PhoneNumberFormField> {
           autofocus: widget.autofocus,
           readOnly: !widget.enabled,
           controller: _internalController,
-          onChanged: widget.onPhoneNumberChanged != null
-              ? (value) => widget.onPhoneNumberChanged!(completePhoneNumber)
-              : null,
+          onChanged: (value) {
+            _onPhoneNumberChanged(value);
+            widget.onPhoneNumberChanged != null
+                ? (v) => widget.onPhoneNumberChanged!(completePhoneNumber)
+                : null;
+          },
           inputFormatters: [
             phoneInputFilter,
             PhoneInputFormatter(
               defaultCountryCode: _selectedPhoneCountry.countryCode,
               allowEndlessPhone: true,
             )
           ],
+          // 98212841
```

Comment:
@IguJl15 cruft? hopefully not personal info?

Reply:
looks good otherwise, just want to make sure we're not merging your phone number

Reply:
Just a random valid phone number from brazil that starts with an country code (`98`: Iran). Srr haha